Dockerfile

```diff
@@ -155,6 +155,8 @@ ENV REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 
 RUN mkdir -p /certs
 
+COPY ./docker/munimap-app/logging.yaml /opt/etc/munimap/configs/logging.yaml
```

Comment:
The `docker` directory is ignored by docker via `.dockerignore`.

Reply:
I moved it to the munimap folder